CredentialPostTest/CredentialPostTest/Pages/Index.cshtml.cs

```diff
@@ -244,7 +239,7 @@ internal class OneTimeCodeRequest
     
     internal class OneTimeCodeResponse
     {
-        [JsonProperty("one_time_code")]
+        [JsonProperty("oneTimeCode")]
```

Comment:
Why is JsonProperty different? Why not name the property `OneTimeCode`?

Reply:
fixed